app/src/main/res/layout/item_plant.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="10dp"
+    android:layout_marginStart="5dp"
+    android:layout_marginEnd="5dp"
+    card_view:cardCornerRadius="10dp"
```

Comment:
1. Same here use 8 instead of 5
2. Also move some sizes that you will reuse into dimens.xml file - for example, some paddings and margins easily could be moved into dimens https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614

Reply:
Or you can look there 

![](https://i.stack.imgur.com/UbsBV.png)